### ISSUE TITLE
Implement Plugin remove

### ISSUE THREAD

**Issue by daniel-vera-g** [k3ai/k3ai#48]
Hey, great work with K3ai. Works For most of the operations pretty smooth.

After experimenting a bit with Kubeflow I wanted to remove a plugin, but it seems that the command is not implemented:

```bash
➜ k3ai  plugin remove --name kf-pa
Remove a given plugin based on NAME

Usage:
  k3ai[options] plugin remove [-n NAME] [other flags]

Flags:
  -n, --name string     NAME of plugin to be created/deleted
  -t, --target string   Target from where to remove plugin.
  -q, --quiet           Suppress output messages. Useful when k3ai is used within scripts.
  -c, --config string   Configure K3ai using a custom config file.[-c /path/tofile] [-c https://urlToFile]
```

See here: https://github.com/k3ai/k3ai/blob/main/cmd/plugin.go#L138

I am not sure, whether  I am missing something but couldn't find anything related in the issues or Roadmap.

---

- Your operating system name and version: Ubuntu 18.04
- Detailed steps to reproduce the bug: Follow exact steps from documentation or README to deploy a plugin

**Comment by alefesta** [comment]
Hi @daniel-vera-g the remove plugin is in the works for 1.0.2 the main issue is that I need to find a standard way to manage the removal of plugins from the various clusters possibly using similar logic as the one used for the installation.
Also, I have to add this to the docs. I'll add this issue to the 1.0.2 roadmap.
Thanks for reporting and happy to see you like to use K3ai!

**Comment by daniel-vera-g** [comment]
> Hi @daniel-vera-g the remove plugin is in the works for 1.0.2 the main issue is that I need to find a standard way to manage the removal of plugins from the various clusters possibly using similar logic as the one used for the installation. Also, I have to add this to the docs. I'll add this issue to the 1.0.2 roadmap. Thanks for reporting and happy to see you like to use K3ai!

Yeah, I also came to the same conclusion when thinking about an implementation.
To be fair, in the docs command reference the `remove` option is not shown for the plugins. Nevertheless, it is in the _cli_  help reference.  
Would it be an option to hide the `remove` command till it's implemented? Maybe that would avoid unnecessary confusion.
